oauth2-authorization-server/spring-security-oauth2-authorization-server.gradle

```diff
@@ -7,6 +7,7 @@ dependencies {
 	compile 'org.springframework.security:spring-security-oauth2-jose'
 	compile springCoreDependency
 	compile 'com.nimbusds:nimbus-jose-jwt'
+	compile 'com.nimbusds:oauth2-oidc-sdk'
```

Comment:
This was added mainly to use [the _TokenIntrospectionSuccessResponse_ class](https://www.javadoc.io/doc/com.nimbusds/oauth2-oidc-sdk/5.0/com/nimbusds/oauth2/sdk/TokenIntrospectionSuccessResponse.html), but it might simply not be worth it. 
Of course, we might need it in the future (If I'm not mistaken the Spring Security Resource Server functionality uses this library as well), but note that it also forces us to handle some Nimbus specific classes (nothing too complicated though) that could be avoided by using our own response implementation.